includes/admin/views/notifications.php

```diff
@@ -0,0 +1,106 @@
+<?php
+/**
+ * Displays a list of notifications.
+ *
+ * @package   easy-digital-downloads
+ * @copyright Copyright (c) 2021, Easy Digital Downloads
+ * @license   GPL2+
+ * @since     2.11.4
+ */
+?>
+<div
+	id="edd-notifications"
+	class="edd-hidden"
+	x-data
+	x-init="function() { $el.classList.remove( 'edd-hidden' ) }"
+>
+	<div
+		class="edd-overlay"
+		x-show="$store.eddNotifications.isPanelOpen"
+		x-on:click="$store.eddNotifications.closePanel()"
+	></div>
+
+	<div
+		id="edd-notifications-panel"
+		x-show="$store.eddNotifications.isPanelOpen"
+		x-transition:enter-start="edd-slide-in"
+		x-transition:leave-end="edd-slide-in"
+	>
+		<div id="edd-notifications-header" tabindex="-1">
+			<h3>
+				<?php
+				echo wp_kses(
+					sprintf(
+					/* Translators: %s - number of notifications */
+						__( '(%s) New Notifications', 'easy-digital-downloads' ),
+						'<span x-text="$store.eddNotifications.numberActiveNotifications"></span>'
+					),
+					array( 'span' => array( 'x-text' => true ) )
+				);
+				?>
+			</h3>
+
+			<button
+				type="button"
+				class="edd-close"
+				x-on:click="$store.eddNotifications.closePanel()"
+			>
+				<span class="dashicons dashicons-no-alt"></span>
+				<span class="screen-reader-text"><?php esc_html_e( 'Close panel', 'easy-digital-downloads' ); ?></span>
+			</button>
+		</div>
+
+		<div id="edd-notifications-body">
+			<template x-if="$store.eddNotifications.notificationsLoaded && $store.eddNotifications.activeNotifications.length">
+				<template x-for="(notification, index) in $store.eddNotifications.activeNotifications" :key="notification.id">
+					<div class="edd-notification">
+						<div class="edd-notification--icon" :class="'edd-notification--icon-' + notification.type">
+							<span class="dashicons" :class="'dashicons-' + notification.icon_name"></span>
+						</div>
+
+						<div class="edd-notification--body">
+							<div class="edd-notification--header">
+								<h4 class="edd-notification--title" x-text="notification.title"></h4>
+
+								<div class="edd-notification--date" x-text="notification.relative_date"></div>
+							</div>
+
+							<div class="edd-notification--content" x-html="notification.content"></div>
+
+							<div class="edd-notification--actions">
+								<template x-for="button in notification.buttons">
+									<a
+										:href="button.url"
+										:class="button.type === 'primary' ? 'button-primary' : 'button'"
```

Comment:
I think the button class needs to be `button button-primary` or `button button-secondary` (the latter may not be as important, but might be safer). On a smaller screen, without the `.button` class, the buttons are two different sizes:

![image](https://user-images.githubusercontent.com/4853066/143030202-ef317930-d18a-4b85-9e6e-5b8406bd1dcc.png)

Adding the `.button` class to the primary one gets them to match. (I'm guessing this is the place where it's set but if I'm wrong...)

Reply:
Yep you got it right! Fixed via https://github.com/awesomemotive/easy-digital-downloads/pull/8941/commits/6ad587db0faae9c2b8c5efc75dc77675b2a541a5